index.html

```diff
@@ -1,40 +1,78 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>What Should We Eat Today?</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Quando|Rosario|Pacifico" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+
     <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
 </head>
+
 <body>
+    <header>
+        <span>Food</span><span>Chooser</span>
+    </header>
     <main>
         <div class="intro">
-            Hi, Welcome to the FoodChooser.
-            May I take your order?
+            <blockquote>
+                <span class="quote">“</span> Hi, Welcome to the FoodChooser.
+                May I take your order? <span class="quote">”</span>
+            </blockquote>
+
+            <hr class="hori">
+
             This app is designed to make choosing what to eat a lot easier, especially when you have a lot of options.
-            It's also helpful for settling arguments when you want to eat one thing but <i>somebody</i> wants to eat somehing else.
-            <ul>How it works:
+            It's also helpful for settling arguments when you want to eat one thing but <i>somebody</i> wants to eat
+            somehing else.
+
+            <br>
```

Comment:
Instead of using the br tag, you can use the margin property in CSS to put space between elements